src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java

```diff
@@ -463,7 +463,7 @@ void testMultipleModificationsWithError() throws Exception {
         assertNotNull(network.getGenerator("idGenerator"));
         assertEquals(20, network.getGenerator("idGenerator").getTargetP(), 0.1);
         assertLogMessage("Technical error: java.lang.NullPointerException: Cannot set property 'targetP' on null object",
-                groovyScriptInfos.getErrorType().name(), reportService);
+                "network.modification.server.errorMessage", reportService);
```

Comment:
Use constant ERROR_MESSAGE_KEY (see remark above in NetworkModificationApplicator)

Reply:
done